source/client-side-encryption/tests/legacy/fle2v2-CreateCollection.yml

```
@@ -35,92 +31,94 @@ tests:
         object: testRunner
         arguments:
           database: *database_name
-          collection: "encryptedCollection"
+          collection: *encrypted_collection_name
           index: __safeContent___1
 
     expectations:
         # events from dropCollection ... begin
         - command_started_event:
             command:
-              drop: "enxcol_.encryptedCollection.esc"
-            command_name: drop
-            database_name: *database_name
-        - command_started_event:
-            command:
-              drop: "enxcol_.encryptedCollection.ecc"
+              drop: *esc_collection_name
             command_name: drop
             database_name: *database_name
         - command_started_event:
             command:
-              drop: "enxcol_.encryptedCollection.ecoc"
+              drop: *ecoc_collection_name
             command_name: drop
             database_name: *database_name
         - command_started_event:
             command:
-              drop: "encryptedCollection"
+              drop: *encrypted_collection_name
             command_name: drop
             database_name: *database_name
         # events from dropCollection ... end
         # events from createCollection ... begin
         # State collections are created first.
         - command_started_event:
             command:
-              create: "enxcol_.encryptedCollection.esc"
-              clusteredIndex: {key: {_id: 1}, unique: true}
-            command_name: create
-            database_name: *database_name
-        - command_started_event:
-            command:
-              create: "enxcol_.encryptedCollection.ecc"
+              create: *esc_collection_name
               clusteredIndex: {key: {_id: 1}, unique: true}
             command_name: create
             database_name: *database_name
         - command_started_event:
             command:
-              create: "enxcol_.encryptedCollection.ecoc"
+              create: *ecoc_collection_name
               clusteredIndex: {key: {_id: 1}, unique: true}
             command_name: create
             database_name: *database_name
         # Data collection is created after.
         - command_started_event:
             command:
-              create: "encryptedCollection"
-              encryptedFields: *encrypted_fields0
+              create: *encrypted_collection_name
+              encryptedFields: &encrypted_fields_expectation {
+                # Expect state collections are not included in the encryptedFields sent to the server.
+                "escCollection": null,
+                "ecocCollection": null,
+                "eccCollection": null,
+                "fields": [
+                    {
+                        "path": "firstName",
+                        "bsonType": "string",
+                        "keyId": { "$binary": { "subType": "04", "base64": "AAAAAAAAAAAAAAAAAAAAAA==" }}
+                    }
+                ]
+            }
```

```suggestion
              encryptedFields: &encrypted_fields_expectation
                # Expect state collections are not included in the encryptedFields sent to the server.
                escCollection: null
                ecocCollection: null
                eccCollection: null
                fields:
                  - path: "firstName"
                    bsonType: "string"
                    keyId: { $binary: { base64: "AAAAAAAAAAAAAAAAAAAAAA==", subType: "04" }}
```

Removing the JSON-like syntax with trailing commas.

---

I do not think rewriting the JSON adds value. The JSON more closely matches the the `encryptedFields` shown in output of `listCollections` from the server. The JSON may be easier for developers to visually compare. I prefer not to change.